libafl/src/events/llmp.rs

```diff
@@ -1439,17 +1425,22 @@ where
                     if child_status == 137 {
                         // Out of Memory, see https://tldp.org/LDP/abs/html/exitcodes.html
                         // and https://github.com/AFLplusplus/LibAFL/issues/32 for discussion.
+                        // Tell the broker that we are exiting
+                        let _ = mgr.send_exiting();
```

Comment:
I don't think we are allowed to use LLMP in this context, since the forked child will have written to the page already (?)

Reply:
You mean we can not use LlmpClient after fork() from the restarter side?

Reply:
Don't think we can, the child may have written a bunch of stuff in the meantime, and it may even have mapped more pages, ..